pkgs/development/python-modules/cysignals/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cython
+, sphinx
+}:
+
+buildPythonPackage rec {
+  pname = "cysignals";
+  version = "1.6.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "003invnixqy1h4lb358vwrxykxzp15csaddkgq3pqqmswnva5908";
+  };
+
+  hardeningDisable = [
+    "fortify"
+  ];
+
+  # currently fails, probably because of formatting changes in gdb 8.0
```

Comment:
If you can, please open an upstream issue about that.

Reply:
Upstream seems aware that there are gdb issues (https://github.com/sagemath/cysignals/pull/52) but generally not very active. Since its a sagemath project, they'll probably fix it once sage updates its gdb.